src/test/java/io/xstefank/wildfly/bot/PROpenedTest.java

```diff
@@ -421,6 +424,61 @@ void testPullRequestFormatTitleCheckOnOpen() throws IOException {
             });
     }
 
+    @Test
+    void testTitleBodyCheckTitleRegexOnOpen() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder("pr-template.json")
+                .title("[WFLY-00000] Metrics PR")
+                .action(Action.OPENED)
+                .build();
+        given().github(mocks -> {
+                    mocks.configFile(RuntimeConstants.CONFIG_FILE_NAME).fromString(
+                            """
+                                wildfly:
+                                  rules:
+                                    - titleBody: "metrics|micrometer"
+                                      notify: [7125767235]
+                                  format:
+                                    commit:
+                                      enabled: false
+                                """);
+                    MockedGHPullRequestProcessor.processEmptyPullRequestMock(mocks.pullRequest(gitHubJson.id()));
+                })
+                .when().payloadFromString(gitHubJson.jsonString())
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
+                    Mockito.verify(mockedPR).comment("/cc @7125767235");
+                });
+    }
+
+
+    @Test
+    void testTitleBodyCheckBodyRegexOnOpen() throws IOException {
```

Comment:
There is a different regex check on EDIT? If not, I would remove the "onOpen" do prevent confusion